### PR TITLE
Remove extra confirmation window

### LINKED ISSUE #68: Remove the verificaiton step from result inputting - if there's a mismatch the system will know - will help tournaments run more smoothly
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by stash86**
fixes #68 

1. For non-league round, if the player reports a result and don't wish to drop, there is no additional confirmation
2. For case number 1 but the player wish to drop, a confirmation pop-up will show to make sure they wish to drop, before processing the result
3. Add a line to catch case when a player hit Submit without choosing a result

### CODE REVIEW

File: gatherling/report.php [hunk 1]
```diff
@@ -10,10 +10,7 @@
     if (isset($_POST['action'])) {
         if ($_POST['action'] == 'finalize_result') {
             // write results to matches table
-            $drop = false;
-            if (isset($_POST['drop'])) {
-                $drop = $_POST['drop'] == 'Y';
-            }
+            $drop = isset($_POST['drop']);
```
Comment (silasary) [inline gatherling/report.php:13]: Why did you remove the value check?

File: gatherling/report.php [hunk 2]
```diff
@@ -292,3 +281,20 @@ function print_verify_resultForm($report, $match_id, $player, $drop, $opponent,
...
+
+<script>
+function verify_input(form) {
+    //Check if the player wants to drop
+    if(form[9].checked) {
+        return confirm('Are you sure you want to drop? This cannot be undone.');
+    }
+
+    //Check if no result being selected
+    if(!form[5].checked && !form[6].checked && !form[7].checked && !form[8].checked) {
```
Comment (silasary) [inline gatherling/report.php]: I don't like using numerical indices here,  It's confusing and could cause issues with a later refactor.

**Review by silasary — CHANGES_REQUESTED** [review]

File: gatherling/report.php (hunk 1 above)
Comment (stash86) [reply gatherling/report.php:13]: Because now you only need to check if the POST['drop'] exist or not. If it exist, means they are dropping. If not exist, means they are not dropping.

Comment (stash86) [reply gatherling/report.php:13]: When the checkbox isn't checked, no 'drop' value will be sent in the POST data.

**Comment by stash86** [comment]
Changes done
